src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -758,21 +782,21 @@ void applyBoosting_(const UInt i,
 		    const Real boost,
 	            vector<Real>& output) {
   if(boost < htm::Epsilon) return; //skip for disabled boosting
-  output[i] = exp((targetDensity - actualDensity[i]) * boost); //TODO doc this code
+  output[i] = exp((targetDensity - actualDensity[i]) * boost); //exponential boosting, default for Numenta
+  //output[i] = log(actualDensity[i]) / log(targetDensity); //log boosting
```

Comment:
What I'd like to do is make the default `boostingStrength = -12345`, and have that be a flag to use the logarithmic boosting function.  This way we can keep the current API and have new code use the logarithmic boosting by default.

Reply:
This is a really good idea! I'll implement it in #545